packages/babel-parser/src/index.js

```diff
@@ -98,3 +98,6 @@ function getParserClass(pluginsFromOptions: PluginList): Class<Parser> {
   }
   return cls;
 }
+
+export default parse;
+export { parse };
```

Comment:
`export { parse as default }` is required to create a live binding:
```suggestion
export { parse as default };
```

Using `export default <expression>` creates a snapshot of the `expression`.